src/redux/reducers/auth.reducer.js

```diff
@@ -42,6 +44,13 @@ export default (state = initialState, action) => {
         ...state,
         verifailed: true,
       };
+    case LOG_OUT_SUCCESS:
+      return {
+        ...state,
+        logOut: true,
+      };
+    case LOG_OUT:
+      return !!sessionStorage.token;
```

Comment:
I'm wondering which state are you updating here

Reply:
@eliemugenzi  when happens when you failed to logut